tests/specs/session-replay/session-state.e2e.js

```diff
@@ -0,0 +1,131 @@
+import { supportsMultipleTabs } from '../../../tools/browser-matcher/common-matchers.mjs'
+import { RRWEB_EVENT_TYPES, config, getSR } from './helpers.js'
+
+/** The "mode" with which the session replay is recording */
+const MODE = {
+  OFF: 0,
+  FULL: 1,
+  ERROR: 2
+}
+
+export default (function () {
+  describe('session manager state behavior', () => {
+    afterEach(async () => {
+      await browser.destroyAgentSession(browser.testHandle)
+    })
+
+    describe('session manager mode matches session replay instance mode', () => {
+      it('should match in full mode', async () => {
```

Comment:
This test passes only about half the time for me on my local, just FYI in case it matters.

Reply:
ty, fixed